content/millennium/bulk-data/group-export.md

```diff
@@ -0,0 +1,296 @@
+
+### Recommendations
+
+* Limit group sizes to around a thousand patients or fewer.
+* Use the `_type` parameter whenever possible to improve response time and minimize storage requirements. `_typeFilter` may also be used to further refine the results to be returned.
+* Use the `_since` parameter to limit the export to only the relevant date range.
+* Clients should follow an Exponential Backoff approach when polling for status as outlined in the [FHIR Bulk Data Access IG](https://hl7.org/fhir/uv/bulkdata/STU1.0.1/export/index.html#bulk-data-status-request).
+* Files should be retrieved promptly as the exported content will expire after 30 days.
+* Once the files have been retrieved, use the delete request to clean up the data from your request.
+
+## Kick Off Request
+
+Kicks off a bulk export request for a group of patients:
+
+    GET /Group/<group_id>/$export
+
+Operation is defined at:
+
+    GET /OperationDefinition/group-export?_format=json
+
+_Implementation Notes_
+
+* Bulk data export requests do not currently honor the Accept-Language header; 'en-US' will be used.
+* Only one bulk export request for a client application and tenant combination may be in-progress at a single time.
+
+### Authorization Types
+
+<%= authorization_types(provider: false, patient: false, system: true) %>
+
+_Notes_
+
+* The authorization token does NOT require the `Group.read` scope.
+* The authorization token must contain system read scopes for all resources requested in the `_type` parameter.
+
+### Parameters
+
+Name                    | Required? | Type        | Description
+------------------------|-----------|-------------|---------------------------------------------------------------------------------------------------
+`_type`                 | No        | [`string`]  | A string of comma-delimited FHIR resource types. Example: `_type=Patient,Encounter,Location`
+`_typeFilter`           | No        | [`string`]  | A string of comma-delimited FHIR Rest queries provided in combination with `_type`. Example: `_typeFilter=Condition?clinical-status=active,Encounter?status=planned`
+`_since`                | No        | [`instant`] | Resources updated after this time will be included in the response. Example: `2021-03-18T19:19:42.000Z`
+`includeAssociatedData` | No        | [`string`]  | When provided, the group export will return or omit a pre-defined set of FHIR resources associated with the request. A string of comma-delimited values. Example: `includeAssociatedData=LatestProvenanceResources`
+`_outputFormat`         | No        | [`string`]  | The format for the requested bulk data files to be generated. Example: `_outputFormat=application/ndjson`
+
```

Comment:
On a second though do we want to make it clear  we support **includeAssociatedData** from v2? as our overview says we implement V1 but includeAssociatedData is from V2

Reply:
> On a second though do we want to make it clear we support **includeAssociatedData** from v2? as our overview says we implement V1 but includeAssociatedData is from V2

I wondered about in the overview after we make the statement that we implement v1 version of the spec, we also mention that we also support several experimental parameters from v2.

Reply:
Added and moved the statement to under Implemented Exports since it made more sense to me to have it there instead of under Registering Client Applications: https://github.com/cerner/fhir.cerner.com/pull/899/commits/c936d35a3ba46f521cf0e5f6addfec2b7276c599#diff-b781e82e47439c1cc68cf3a6a51ab1668b5f40d37870022bec1936c5c69d6bc1R39